apps/random_pipeline/bench_comparison.sh

```diff
@@ -17,7 +17,7 @@ SCHEDULES=1
 RANDOM_DROPOUT=100
 BEAM_SIZE=50
 
-make -C ../autoscheduler bin/libauto_schedule.so
+make -C ../autoscheduler ../autoscheduler/bin/libauto_schedule.so
```

Comment:
This change looks fishy to me. make -C already sets the working directory to its argument.

Reply:
Will fix.